src/app/modules/account-detail/containers/account-detail/account-detail.component.ts

```diff
@@ -18,7 +28,7 @@ export class AccountDetailComponent implements OnInit {
   defaultPerPageOptions = defaultPerPageOptions;
   defaultSortBy = defaultSortBy;
 
-  public recipes: Recipe[] = [];
+  public recipes: Observable<Recipe[]> = this.store.select(selectRecipes);
```

Comment:
Move `= this.store.select(selectRecipes)` to `ngOnInit`
```suggestion
  public recipes: Observable<Recipe[]>;
```